sdk/storage/src/table/mod.rs

```diff
@@ -1,14 +1,29 @@
-mod batch;
-mod cloud_table;
-mod continuation_token;
-pub mod de;
-pub mod paginated_response;
-pub mod table_client;
-mod table_entity;
-pub use batch::*;
-pub use cloud_table::*;
-pub use continuation_token::ContinuationToken;
-pub use paginated_response::PaginatedResponse;
-pub use table_client::*;
-pub use table_entity::*;
+pub mod clients;
+mod entity_metadata;
+mod entity_with_metadata;
+mod filter;
+mod if_match_condition;
+pub mod prelude;
 pub mod requests;
+pub mod responses;
+mod return_entity;
+mod select;
+mod table;
+mod top;
+mod transaction;
+mod transaction_operation;
+pub use entity_metadata::EntityMetadata;
+pub use entity_with_metadata::EntityWithMetadata;
+pub use filter::Filter;
+pub use if_match_condition::IfMatchCondition;
+pub use return_entity::ReturnEntity;
+pub use select::Select;
+pub use table::Table;
+pub use top::Top;
+pub use transaction::Transaction;
+pub use transaction_operation::TransactionOperation;
+
+// we need this since the http::Method does not have the MERGE verb. The unwrap is safe here.
+lazy_static! {
```

Comment:
FYI: If you've not used the `OnceCell` crate, you might want to give it a try. It is likely going to be what gets merged into the std library as the standard way to handle lazy evaluation.

Reply:
I was unaware of it. I will change the code to use OnceCell, seems pretty straightforward. 👍 Thanks to pointing it out!